src/components/Post/MoreText.tsx

```diff
@@ -0,0 +1,27 @@
+import { Collapse, CollapseProps, Button, useBoolean } from '@chakra-ui/react';
+
+type MoreTextProps = CollapseProps & {
+  text: string;
+  maxHeight: number;
+};
+
+const MoreText = ({ text, maxHeight, ...props }: MoreTextProps) => {
+  const [toggle, setToggle] = useBoolean(false);
```

Comment:
```suggestion
  const [istoggled, setIsToggled] = useBoolean();
```
```suggestion
  const [toggle, setToggle] = useBoolean(false);
```
`useBoolean` 기본값은 `false`입니다!!
또한 이름은 직관적으로 위와 같이 변경하는게 나을 것 같아요!!